src/ClapCommandLineHandler.package/ClapHandlerCollectorStrategy.class/instance/collectHandlers.st

```diff
@@ -0,0 +1,3 @@
+as yet unclassified
+collectHandlers
+	^ SubclassResponsibility
```

Comment:
This is not how you use that :)

This class is an exception, you're supposed to signal (raise) it. Usually this is done by using `self subclassResponsibility` (message implemented by `Object` as the method body.